aptos-move/move-examples/post_mint_reveal_nft/sources/minting.move

```diff
@@ -17,13 +17,35 @@
 // We choose to implement the two-collection method as a tutorial because it's simple and hands-off (you don't have to manually update
 // anything after setting up the collection).
 //
-// The flow will be:
-// 1. Publish this contract using a resource account
-// 2. Set up the collection by calling `set_admin`, `set_treasury`, `set_collection_config_and_create_collection`, `
-// set_minting_and_reveal_config`, `add_to_whitelist`, `add_tokens` functions.
-// 3. When minting starts, call `mint_source_certificate` to mint a source certificate token.
-// 4. When reveal starts, call `exchange` to exchange a source certificate token to a destination token.
-// You can refer to the unit tests below `set_up_test` and `test_happy_path` as examples of the expected flow.
+// Now let's go over the flow using a 2-stage whitelist NFT project as an example:
+// We will use two accounts to set up the project: a source_account and an admin_account.
+// Note that they can be the same account, depending on your use case.
+// ##### Setup #####
+// 1. Publish this contract
+// aptos move publish --named-addresses post_mint_reveal_nft=[address of your source_account]
+// 2. Use the source account to set up the collection's admin (the account that has the right to set up the rest of the account)
```

Comment:
Why not just make the source account the admin account? This is very confusing

Reply:
good point, updating now